astropy/io/votable/tests/test_table.py

```diff
@@ -52,6 +52,49 @@ def home_is_tmpdir(monkeypatch, tmp_path):
     monkeypatch.setenv("USERPROFILE", str(tmp_path))
 
 
+def test_read_write_votable_parquet(tmp_path):
+    """
+    Test to write VOTable with Parquet serialization
+    """
+
+    # Create some fake data
+    number_of_objects = 10
+    ids = [f"COSMOS_{ii:03g}" for ii in range(number_of_objects)]
+    redshift = np.random.uniform(low=0, high=3, size=number_of_objects)
+    mass = np.random.uniform(low=1e8, high=1e10, size=number_of_objects)
+    sfr = np.random.uniform(low=1, high=100, size=number_of_objects)
+    astropytab = Table([ids, redshift, mass, sfr], names=["id", "z", "mass", "sfr"])
+
+    # Create Column metadata
+    column_metadata = {
```

Comment:
Can a check be added that the column metadata are read back in?

Reply:
done